presto-parser/src/main/java/com/facebook/presto/sql/parser/warnings/ParsingWarning.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.parser.warnings;
+
+import com.facebook.presto.sql.parser.ParsingException;
+import com.facebook.presto.sql.tree.NodeLocation;
+import org.antlr.v4.runtime.RecognitionException;
+
+public class ParsingWarning
+        extends ParsingException
```

Comment:
Do we want to extend an exception? It will trigger the stacktrace collection every time we create an instance of this class. Do we really need that?